src/Database/PG/Query/Connection.hs

```diff
@@ -348,41 +341,6 @@ assertResCmd conn mRes = do
     checkResOk (ResultOkData _) = throwPGIntErr $
       PGIUnexpected "cmd expected; tuples found"
 
--- These are convenient wrappers around LibPQ's similar functions
-{-# INLINE prepare' #-}
-prepare'
-  :: PQ.Connection  -- ^ connection
-  -> RemoteKey      -- ^ stmtName
-  -> Template       -- ^ query
-  -> [PQ.Oid]       -- ^ paramTypes
-  -> PGExec ()      -- ^ result
-prepare' conn rk (Template t) ol = do
-  mRes <- liftIO $ PQ.prepare conn rk t $ Just ol
-  assertResCmd conn mRes
-
-{-# INLINE execPrepared #-}
-execPrepared
-  :: PQ.Connection                -- ^ connection
-  -> [Maybe (DB.ByteString, PQ.Format)]           -- ^ parameters
-  -> RemoteKey                    -- ^ stmtName
-  -> PGExec ResultOk -- ^ result
-execPrepared conn args n = do
-  mRes <- lift $ PQ.execPrepared conn n args PQ.Binary
-  checkResult conn mRes
```

Comment:
Should we just keep this rather than manually inlining it twice, but just not have it exported?

Reply:
Thanks for your review!

I think it's only inlined once? We had `execParams` and `execPrepared`, which were both called only from `execQuery`.

Reply:
Fair enough, I missed the part where the two apparent calls were actually subtly different by calling `execParams` vs `execPrepared`. :+1: